acasclient/acasclient.py

```diff
@@ -9,10 +9,15 @@
 from pathlib import PurePath
 import re
 import base64
+from io import StringIO
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
+VALID_SEARCH_TYPES = {"substructure", "duplicate",
```

Comment:
nit: I think this should be `VALID_STRUCTURE_SEARCH_TYPES` to be specific about what kind of search